src/vit-servicing-station/vit-servicing-station-lib/src/db/queries/funds.rs

```diff
@@ -84,15 +99,20 @@ pub struct FundNextInfo {
 pub async fn query_current_fund(pool: &DbConnectionPool) -> Result<FundWithNext, HandleError> {
     let db_conn = pool.get().map_err(HandleError::DatabaseError)?;
     tokio::task::spawn_blocking(move || {
-        let funds: Vec<Fund> = fund_dsl::funds
-            // TODO: Not sure if sorting by the PK is actually necessary
-            //
-            // this assumes that the next will be the second inserted
-            // and that the current is the first.
-            .order(fund_dsl::id)
-            .limit(2)
-            .load(&db_conn)
-            .map_err(|_e| HandleError::NotFound("fund".to_string()))?;
+        let db_conn = &db_conn;
+
+        let funds: Vec<Fund> = q!(
+            db_conn,
+            fund_dsl::funds
+                // TODO: Not sure if sorting by the PK is actually necessary
```

Comment:
off topic; it would be nice to start removing these TODOs if they are no longer relevant.

Reply:
I agree. Didn't touch them because I'm still not that familiar with the code and don't know whether they can be removed.